postgres/init/22-portal-content.sql

```diff
@@ -86,6 +86,8 @@ aa0a0000-7fbc-1f2f-817f-bce0502c1015	1	2022-09-18 00:00:00+00	2022-09-18 00:00:0
 aa0a0000-7fbc-1f2f-817f-bce0502c1016	2	2022-09-18 00:00:00+00	2022-09-18 00:00:00+00	\N	Marketplace Terms & Conditions	ac861325-bc54-4583-bcdc-9e9f2a38ff84	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
 aa0a0000-7fbc-1f2f-817f-bce0502c1017	4	2022-09-18 00:00:00+00	2022-09-18 00:00:00+00	\N	Terms & Conditions Service Provider	2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
 aa0a0000-7fbc-1f2f-817f-bce0502c1018	4	2022-09-18 00:00:00+00	2022-09-18 00:00:00+00	\N	Data Sharing Approval - allow CX to submit company data (company name, requester) to process the subscription	2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
+aa0a0000-7fbc-1f2f-817f-bce0502c1090	1	2022-01-01 00:00:00.388+00	2022-01-01 00:00:00.388+00	\N	I confirm that my company has successfully received a Catena-X onboarding certificate issued by an official Conformity Assessment Body (CAB). I awknowledge to upload the certificate. For more information please visit the association homepage (Link)  2dc4249f-b5ca-4d42-bef1-7a7a950a4f87	1aacde78-35ec-4df3-ba1e-f988cddcbbd9
```

Comment:
The text is still to long, max allowed would be:

_I confirm that my company has successfully received a Catena-X onboarding certificate for application providers / service providers issued by an official Catena-X Conformity Assessment Body (CAB). I acknowledge that I am able to upload the certificate in _

Reply:
@Phil91 did you received that error with an upload test? asking cause I have tested the sentence earlier and 255 alphanumeric characters are allowed and the agreement name has 248 characters.
I have also been able to successfully load the name inside the dev db, I am surprised that it fails for you

Reply:
ah maybe it was a space/tab issue - looks now better I believe